src/app/services/shared.service.ts

```diff
@@ -7,4 +7,5 @@ export class SharedService {
   public isAdmin: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
   public isServiceUp: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
   public showLogs: BehaviorSubject<any> = new BehaviorSubject<any>(false);
+  public theme: BehaviorSubject<any> = new BehaviorSubject<any>(localStorage.getItem('OPTED_THEME'));
```

Comment:
There is no need to read value from localStorage. Put default value here that otherwise it will be blank on first load until no value saved in localStorage. Also it defined the default behaviour only here.